go/client/cmd_simplefs.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"context"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+// NewCmdDevice creates the device command, which is just a holder
+// for subcommands.
+func NewCmdSimpleFS(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "simplefs",
```

Comment:
@gabriel is it safe to nuke that `keybase fs` command?  doesn't seem to have many features behind it.